solidity/random-beacon/test/system/e2e.test.ts

```diff
@@ -51,53 +59,46 @@ const fixture = async () => {
 // new signing group. The next steps call the random beacon relay requests and
 // validate the results of the submitted signatures. At the end of this scenario
 // 3 active groups should be added to the chain as a result of signatures submission
-// and dkg under the hood. All the init params map 1:1 real params set in the
+// and dkg under the hood.
+// Group creation frequency is adjusted for the purpose of the testing data.
+// All the other init params should be intact and use existing params from the
```

Comment:
Is it true though? This test uses the fixture and the fixture modifies the parameters:

https://github.com/keep-network/keep-core/blob/39cfc613e4fe9894160cb40ac4c884623da8259c/solidity/random-beacon/test/fixtures/index.ts#L103

Reply:
you're right, we use fixtures here. I changed the description [9e5c29f](https://github.com/keep-network/keep-core/pull/3018/commits/9e5c29f8d64f4d1e2c2e7fdfedc9882b389dce24)